docs/_includes/common/page-notice.html

```diff
@@ -43,89 +43,60 @@ <h3 class="page-notice__title">Title copy goes here</h3>
                 <div class="page-notice__footer">
-                    <a href="https://www.ebay.com" class="fake-btn fake-btn--secondary fake-btn--transparent">Action</a>
-                </div>
-            </section>
-            <section class="page-notice page-notice--celebration" role="region" aria-label="Success">
-                <div class="page-notice__header">
-                    <svg class="icon icon--confirmation-filled" focusable="false" height="24" width="24" role="img" aria-label="Success">
-                        {% include common/symbol.html name="confirmation-filled" %}
-                    </svg>
-                </div>
-                <div class="page-notice__main">
-                    <h3 class="page-notice__title">Title copy goes here</h3>
-                    <p>Details...</p>
-                </div>
-                <div class="page-notice__footer">
-                    <a href="https://www.ebay.com" class="fake-btn fake-btn--secondary fake-btn--transparent">Action</a>
+                    <a href="https://www.ebay.com">Action</a>
                 </div>
             </section>
         </div>
     </div>
     {% highlight html %}
-<section class="page-notice page-notice--confirmation" role="region" aria-label="Confirmation">
-    <div class="page-notice__header">
-        <svg class="icon icon--confirmation-filled" focusable="false" height="24" width="24" role="img" aria-label="Confirmation">
-            <use xlink:href="#icon-confirmation-filled"></use>
-        </svg>
-    </div>
-    <div class="page-notice__main">
-        <h3 class="page-notice__title">Title copy goes here</h3>
-        <p>Details...</p>
-    </div>
-    <div class="page-notice__footer">
-        <a href="https://www.ebay.com" class="fake-btn fake-btn--secondary fake-btn--transparent">Action</a>
-    </div>
-</section>
-
-<section class="page-notice page-notice--attention" role="region" aria-label="Attention">
-    <div class="page-notice__header">
-        <svg class="icon icon--attention-filled" focusable="false" height="24" width="24" role="img" aria-label="Attention">
-            <use xlink:href="#icon-attention-filled"></use>
-        </svg>
-    </div>
-    <div class="page-notice__main">
-        <h3 class="page-notice__title">Title copy goes here</h3>
-        <p>Details...</p>
-    </div>
-    <div class="page-notice__footer">
-        <a href="https://www.ebay.com" class="fake-btn fake-btn--secondary fake-btn--transparent">Action</a>
-    </div>
-</section>
+    <section class="page-notice page-notice--confirmation" role="region" aria-label="Confirmation">
+        <div class="page-notice__header">
+            <svg class="icon icon--confirmation-filled" focusable="false" height="24" width="24" role="img" aria-label="Confirmation">
+                {% include common/symbol.html name="confirmation-filled" %}
+            </svg>
+        </div>
+        <div class="page-notice__main">
+            <p>Title copy goes here</p>
```

Comment:
Is there a reason for dropping the heading tag? It should probably remain as a heading.